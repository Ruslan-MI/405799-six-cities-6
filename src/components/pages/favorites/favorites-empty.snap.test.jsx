import React from "react";
import {
  render,
} from "@testing-library/react";
import FavoritesEmpty from "./favorites-empty";


it(`Favorites empty should render correctly`, () => {
  const {
    container,
  } = render(<FavoritesEmpty />);

  expect(container).toMatchSnapshot();
});
