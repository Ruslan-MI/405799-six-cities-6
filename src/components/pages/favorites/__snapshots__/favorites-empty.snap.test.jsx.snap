// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favorites empty should render correctly 1`] = `
<div>
  <h1
    class="visually-hidden"
  >
    Favorites (empty)
  </h1>
  <div
    class="favorites__status-wrapper"
  >
    <b
      class="favorites__status"
    >
      Nothing yet saved.
    </b>
    <p
      class="favorites__status-description"
    >
      Save properties to narrow down search or plan your future trips.
    </p>
  </div>
</div>
`;
